dj/construction/build.py

```diff
@@ -708,10 +753,22 @@ def build_metric_nodes(
         else columns[0]
         for col_name, columns in dimension_grouping.items()
     ]
+
     combined_ast.select.projection.extend(dimension_columns)
 
-    if organization is not None:
-        combined_ast.organization = organization
+    # go through the orderby items and make sure we put them in the order the user requested them in
+    orderby_sort_items_resorted = orderby_sort_items + [  # type: ignore
```

Comment:
I might be missing something but why is `[None] * len(orderby_mapping)` needed? Won't the `orderby_sort_items_resorted.insert` further down take care of putting the element in the right location? And then the filtering out of `None` can also be removed